### ISSUE TITLE
how to set a timeout for seesion insert

### ISSUE THREAD

**Issue by flyly0755** [xzkostyan/clickhouse-sqlalchemy#188]
```python
from clickhouse_sqlalchemy import make_session
from sqlalchemy import create_engine
from sqlalchemy.pool import NullPool
engine = create_engine(uri, echo=False, poolclass=NullPool)
session = make_session(engine)
```
i want to know, whether clickhouse-sqlalchemy package supports sql cmd execute timeout configuration? and how to config sql insert timeout in  create_engine method?
now i meet situation in this issue link:
https://github.com/xzkostyan/clickhouse-sqlalchemy/issues/186

```python
session.bulk_insert_mappings(table, datalist)
```
with executing bulk_insert_mappings this method may take a long time, and sometimes the clickhouse server side is something wrong and without response to this bulk_insert_mappings  request.
so i want to give a timeout to this bulk_insert_mappings language, so if timeout, this method execution stops and raise some error info.
so how to config, with connection timeout, the right writing is add connect_args={'timeout': 600} this part:
```python
create_engine(dbsession.bind.url, echo=False,
                                          **connect_args={'timeout': 600}**, poolclass=NullPool)
```
with this stackoverflow link:
https://stackoverflow.com/questions/44936527/configure-query-command-timeout-with-sqlalchemy-create-engine

seems config like this: connect_args={"options": "-c statement_timeout=1000"}
but i take a try, it did not work.
Does clickhouse-sqlalchemy support? and how to config?

**Comment by AnVuTrong** [comment]
I don't think this project gonna see this problem anytime soon. The asynch module does support Pool management, but i'm still finding any support for pool management with clickhouse-sqlalchemy
